src/samples/DX11/DX11.cpp

```diff
@@ -356,7 +396,7 @@ int main()
                 const RenderTarget& target = renderTargets.at(description.handle);
                 context->OMSetRenderTargets(1, target.view.GetAddressOf(), nullptr);
 
-                const float clearColour[4] = { 0.f, 0.f, 0.f, 0.f };
+                const float clearColour[4] = { 0.f, 0.2f, 0.f, 0.f };
```

Comment:
Not sure about the green tint. May confuse people with alpha/premult alpha blending errors. 

This should be a clear sample of how to do frontplate rendering.